google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py

```diff
@@ -106,9 +106,11 @@ class StreamingPullManager(object):
             ``projects/{project}/subscriptions/{subscription}``.
         flow_control (~google.cloud.pubsub_v1.types.FlowControl): The flow
             control settings.
-        use_legacy_flow_control (bool): Disables enforcing flow control settings
-            at the Cloud PubSub server and uses the less accurate method of only
-            enforcing flow control at the client side.
+        use_legacy_flow_control (bool):
+            If set to ``True``, it disables enforcing flow control settings on the
+            Cloud Pub/Sub server. Defaults to ``False``.
+            The less accurate method of enforcing flow control on the client side
```

Comment:
I don't know if I'd call it "less accurate." The flow control limits will still be enforced the same way with regard to the user callback. The difference is that there will be fewer messages buffered in the client library itself.

Reply:
The "less accurate" bit was added [here](https://github.com/googleapis/python-pubsub/pull/231/files), but it's indeed a bit odd - being less accurate would imply that there are bugs in the client flow control logic. I'll remove that.